README.md

```diff
@@ -6,6 +6,36 @@ Fissile converts existing BOSH dev releases into docker images.
 It does this using just the releases, without a BOSH deployment, CPIs, or a BOSH
 agent.
 
+## Build and Install
+Building fissile needs Go 1.4 or higher. You can download it from [website](https://golang.org/doc/install)
+   
+### Build procedure
+Execute the following commands to compile fissile
+   
+```
+$ cd $GOPATH
+$ mkdir -p src
```

Comment:
`mkdir -p` creates parent dirs as needed, so only `mkdir -p src/github.com/hpcloud` is required.